play-services-base-core-ui/src/main/res/values-it/strings.xml

```diff
@@ -9,5 +9,4 @@
 
     <string name="list_no_item_none">Nessuna</string>
     <string name="list_item_see_all">Mostra tutte</string>
-    <string name="open_app">Apri</string>
```

Comment:
I think this is an oversight, could you please revert this small change?

Reply:
Done